Rerun dependency chain through rfd->ashpd->zbus->zvariant broken with recent z* releases

**Describe the bug**

Doing cargo update in a package/workspace results in a broken dependency chain that causes compile failures.

**To Reproduce**

1. Depend on any recent version of rerun
2. Run `cargo update` on your toml file
3. Watch it upgrade a dependency on `zbus` from `5.12`->5.13` (also zvariant)
4. Try and build or run clippy on your package, you should see something like:

```
Checking ashpd v0.11.1 (https://github.com/stonier/ashpd.git?branch=patch%2F0_11_1#c0d108ff)
error[E0277]: the trait bound `AppID: Basic` is not satisfied
   --> /opt/cargo/git/checkouts/ashpd-990e4ebe0843e308/c0d108f/src/documents/mod.rs:391:16
    |
391 |         self.0.call("Info", &(doc_id.into())).await
    |                ^^^^ the trait `Basic` is not implemented for `AppID`
    |
    = help: the following other types implement trait `Basic`:
              &B
              BusName<'_>
              ErrorName<'_>
              InterfaceName<'_>
              MemberName<'_>
              NonZero<i16>
              NonZero<i32>
              NonZero<i64>
            and 35 others
    = note: required for `HashMap<AppID, Vec<Permission>>` to implement `zbus::zvariant::Type`
    = note: 1 redundant requirement hidden
    = note: required for `(file_path::FilePath, HashMap<AppID, Vec<Permission>>)` to implement `zbus::zvariant::Type`
```

**Expected behavior**

No failures.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04

**Rerun version**
Rerun 0.23.4

**Additional context**

The old dependency chain is: `re_tracing -> rfd-0.15.*->ashpd-0.11.*->zbus-5.*->zvariant-*`.
The problem seems to have occurred as zbus upgraded from `5.12->5.13` (also zvariant). A cargo update will accept those upgrades, but `ashpd` will fall over on them.

* Building with a [patched version of ashpd](https://github.com/stonier/ashpd/commit/9a58729c5323d14c5e97aeeaccb822699e67c536) fixes the problem
* Newer versions of `ashpd` have already fixed the problem: [ashpd@7a30101](https://github.com/bilelmoussaoui/ashpd/commit/7a3010aba2baf4e84dbef66d9d9df5ca3c149e8c) which is released in `0.12.*`.
* Newer versions of `rfd` do not, in fact, seem to depend on `ashpd` at all. See [0.15.4](https://crates.io/crates/rfd/0.15.4/dependencies) vs [0.17.2](https://crates.io/crates/rfd/0.17.2/dependencies)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -340,7 +340,7 @@ raw-window-handle = "0.6.2"
 rayon = "1.11"
 regex-lite = "0.1.7"
 rexif = "0.7.5"
-rfd = { version = "0.15.4", default-features = false, features = ["async-std", "xdg-portal"] }
+rfd = { version = "0.17.2", default-features = false, features = ["xdg-portal"] }
 ron = { version = "0.11.0", features = ["integer128"] }
 roxmltree = "0.20.0"
 rustdoc-json = "0.9.7"

